ykutil/src/addr.rs

```diff
@@ -43,3 +43,25 @@ pub fn code_vaddr_to_off(vaddr: usize) -> Option<(&'static CStr, u64)> {
     }
     None
 }
+
+#[cfg(test)]
+mod tests {
+    use super::code_vaddr_to_off;
+    use libc::dlsym;
+    use std::ptr;
+
+    #[test]
+    fn map_libc() {
+        let vaddr = unsafe { dlsym(ptr::null_mut(), "getuid".as_ptr() as *const i8) };
```

Comment:
Is `"getuid".as_ptr()` equivalent to C string? I think we should use `CString` to ensure a null terminator?

Reply:
You are right. This is incorrect.